### DISCUSSION TITLE
Guides / Tutorial Feedback and Requests

### DISCUSSION THREAD

**Discussion by raksiv** [nitrictech/nitric#278] [Q&A]
A discussion for feedback or tutorial requests from the community - lets create some content!

**Comment by elisselockhart** [comment]
We've heard from a few folks recently that they're interested in learning more about the abstraction layer and how that works.

**Reply by raksiv** [reply]
We can probably create a post about how the abstraction layer works and its merits.

It might also be worthwhile thinking about content that demonstrates how to interpret the abstraction and implement SDKs.
